worker/src/main/java/org/apache/celeborn/service/deploy/worker/storage/MapPartitionDataWriter.java

```diff
@@ -40,19 +40,19 @@
 /*
  * map partition file writer, it will create index for each partition
  */
-public final class MapPartitionDataWriter extends PartitionDataWriter {
+public class MapPartitionDataWriter extends PartitionDataWriter {
   private static final Logger logger = LoggerFactory.getLogger(MapPartitionDataWriter.class);
 
-  private int numSubpartitions;
-  private int currentDataRegionIndex;
-  private boolean isBroadcastRegion;
-  private long[] numSubpartitionBytes;
-  private ByteBuffer indexBuffer;
-  private int currentSubpartition;
-  private long totalBytes;
-  private long regionStartingOffset;
-  private FileChannel indexChannel;
-  private volatile boolean isRegionFinished = true;
+  protected int numSubpartitions;
```

Comment:
Why does this change all privates to protected? If other needs, please add the getter of the variable.